cmd/login/cmd.go

```diff
@@ -161,8 +172,16 @@ func run(cmd *cobra.Command, argv []string) {
 	// Check mandatory options:
 	env := args.env
 	if env == "" {
-		r.Reporter.Errorf("Option '--env' is mandatory")
-		os.Exit(1)
+		return fmt.Errorf("Option '--env' is mandatory")
+	}
+
+	// Fail fast if RH_KEYRING is provided and invalid
+	keyring := os.Getenv("RH_KEYRING")
```

Comment:
@tylercreller Shall we introduce a `const` for `RH_KEYRING` as we use it quite a bit.

Reply:
Updated